internal/resource/cluster_model.go

```diff
@@ -56,6 +56,10 @@ type cluster struct {
 // }
 
 func (c *cluster) TagsAttribute(ctx context.Context, d diag.Diagnostics) []*console.TagAttributes {
+	if c.Tags.IsNull() {
+		return nil
+	}
```

Comment:
@michaeljguarino I think this should solve the thing you mentioned in the comment above.

Reply:
Ok awesome